incubator/hnc/e2e/tests/issues_test.go

```diff
@@ -0,0 +1,39 @@
+package test
+
+import (
+	"time"
+
+	. "github.com/onsi/ginkgo"
+)
+
+var _ = Describe("Issues", func() {
+
+	const (
+		nsParent = "issues-parent"
+		nsChild = "issues-child"
+	)
+
+	AfterEach(func() {
```

Comment:
_Also_ do this in `BeforeEach` in case the last run was interrupted? E.g. I see a failure, I hit "ctrl-c" but the next test is already running and has created namespaces. But because I hit ctrl-c, `AfterEach` never gets run, and the test fails the next time I run it.

This is a general comment for all e2e tests.

Reply:
Done. will put up another PR to apply this to other tests